kpm/lib/kpm/inspector.rb

```diff
@@ -75,7 +75,14 @@ def build_plugin_map(plugin_name, plugin_path, type)
       plugin_map = { plugin_name: plugin_name, plugin_path: plugin_path.to_s, type: type }
       entries = get_entries(plugin_path)
       set_default = entries.select { |e| e == 'SET_DEFAULT' }[0]
-      default_version = File.basename(File.readlink(plugin_path.join(set_default))) if set_default
+      # On Windows, SET_DEFAULT is a directory, not a symbolic link
+      if set_default
```

Comment:
I would have `default_version = nil` before the `if`, that way the variable is explicitly initialized regardless of the codepath.

In the rescue block, you could put the comment to make it explicit we're ignoring the error on Windows only:
```
        rescue StandardError
          # Ignore: on Windows, SET_DEFAULT is a directory, not a symbolic link
        end
```